src/index.html

```diff
@@ -5,9 +5,45 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
```

Comment:
Fix it if you have only two attributes
```suggestion
    <article class="card" data-qa="card">
```